index.html

```diff
@@ -4372,6 +4372,154 @@ <h2>Revision History</h2>
+  &quot;cryptosuite&quot;: &quot;jcs-eddsa-2022&quot;,
+  &quot;created&quot;: &quot;2023-03-05T19:23:24Z&quot;,
+  &quot;proofPurpose&quot;: &quot;assertionMethod&quot;,
+  &quot;verificationMethod&quot;: pubCEO,
+  &quot;proofValue&quot;: signature(privCEO, document)
+}
+      </code></pre>
+      <p>
+Where <em>pubCEO</em> is used as a placeholder for a reference that resolves to
+the CEO's public key and <code>signature(priv, doc)</code> denotes the
+computation of a digital signature by a particular data integrity cryptosuite
+using a particular private key over a particular document. The <em>type</em>,
+<em>cryptosuite</em>,  <em>created</em>, and <em>proofPurpose</em> attributes do
+not factor into our discussion so we will omit them. In particular below we show
+all the proofs in a <em>proof set</em> on a document that has been signed by the
+VP of engineering, CFO, and CEO:
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will verify
+  &quot;proof&quot;: [
+  {&quot;verificationMethod&quot;: pubVPE, &quot;proofValue&quot;: signature(privVPE, document)},
+  {&quot;verificationMethod&quot;: pubCFO, &quot;proofValue&quot;: signature(privCFO, document)},
+  {&quot;verificationMethod&quot;: pubCEO, &quot;proofValue&quot;: signature(privCEO, document)}]
+}
+      </code></pre>
+      <p>
+The <em>holder</em> or any other intermediary receiving a signed VC containing a
+<em>proof set</em> is able to remove any of the <em>proofs</em> within the set
+prior to passing it on to another entity and this VC will still verify. This may
+or may not be the intent. For the signatories sending a birthday card to a
+valued employee this is probably fine. If we are trying to model a business
+process where approvals ascend the company hierarchy this would not be ideal
+since any intermediary could remove signatures from the <em>proof set</em> and
+still have it verify, i.e., below it looks like the CFO and CEO just approved
+something without the VP of engineering's concurrence.
+      </p>
+      <pre><code class="language-javascript">
+{ // Rest of VC document not shown, this proof set will also verify
+  &quot;proof&quot;: [
+  {&quot;verificationMethod&quot;: pubCFO, &quot;proofValue&quot;: signature(privCFO, document)},
+  {&quot;verificationMethod&quot;: pubCEO, &quot;proofValue&quot;: signature(privCEO, document)}]
+}
+      </code></pre>
+      <p>
+We can introduce a &quot;dependency relation&quot; between <em>proofs</em> in a
+<em>proof set</em> by assigning the proof that another proof relies on, i.e.,
+the <em>dependent proof</em>, an <em>id</em> and the <em>relying proof</em> will
+have a list (or single item) denoting <em>previousProofs</em>. Such
+<em>dependency chains</em> can have arbitrary depth. The <strong>intent</strong>
+of such a <em>proof chain</em> is to model an approval chain in a business
+process or a notary witnessing analog signatures. In such cases we CANNOT allow
```

Comment:
Avoid uppercase "CANNOT", makes it seem like this is normative text when it isn't.